dubbo-monitor/dubbo-monitor-api/src/main/java/org/apache/dubbo/monitor/support/MonitorFilter.java

```diff
@@ -45,15 +45,28 @@
 
     private static final Logger logger = LoggerFactory.getLogger(MonitorFilter.class);
 
+    /**
+     * The Concurrent counter
+     */
     private final ConcurrentMap<String, AtomicInteger> concurrents = new ConcurrentHashMap<String, AtomicInteger>();
 
+    /**
+     * The MonitorFactory
+     */

```

Comment:
Will it be just a MonitorFactory as part of documentation or you have missed it?

Reply:
Let's improve it later.